src/app.rs

```diff
@@ -355,6 +371,33 @@ impl Application for AppModel {
                     .height(Length::Fill)
                     .into()
             }
+            Some(Page::Motherboard) => {
+                let Some(hostnamectl) = &self.hostnamectl else {
+                    return widget::text::title1(fl!("error-occurred")).into();
+                };
+
+                if let Some(hostnamectl_str) = &self.hostnamectl {
+                    if hostnamectl_str.starts_with(fl!("error-occurred").as_str()) {
+                        return widget::text::title1(hostnamectl_str).into();
+                    } else {
+                        let hostnamectl = hostnamectl
+                            .lines()
+                            .map(|line: &str| {
+                                let (prefix, suffix) = line.split_once(':').unwrap();
+                                settings::item(prefix, widget::text::body(suffix)).into()
```

Comment:
```suggestion
                                settings::item(prefix.trim(), widget::text::body(suffix)).into()
```

We should trim all whitespaces out of the prefix, otherwise it looks strange.